proto/cosmos/base/bytes.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package cosmos.crypto.secp256r1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/types";
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.stringer_all)         = false;
+
+// PBBytes defines a structure wrapping protocol buffers raw bytes serialization
+message PBBytes {
```

Comment:
We can just use https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.BytesValue

Reply:
nice, I will try it.

Reply:
Just wondering why is this even needed?

Reply:
for proto marshaling.